.travis.yml

```diff
@@ -5,8 +5,6 @@ services:
 - docker
 deploy:
   on:
-    branches:
-      only:
-        - master
+    branch: release
```

Comment:
I think this should be:
branch: master
tags: true

Reply:
Switched.